graphql_compiler/query_formatting/common.py

```diff
@@ -21,50 +21,51 @@
 # Public API
 ######
 
+def _raise_invalid_type_error(name, expected_python_type_name, value):
+    """Raise a GraphQLInvalidArgumentError that states that the argument type is invalid."""
+    raise GraphQLInvalidArgumentError(u'Invalid type for argument {}. Expected {}. Got value {} of '
+                                      u'type {}.'.format(name, expected_python_type_name, value,
+                                                         type(value).__name__))
 
-def validate_argument_type(expected_type, value):
+
+def validate_argument_type(name, expected_type, value):
```

Comment:
The next version bump is going to be a major version bump. We might as well make the name parameter required here.